src/components/LegacyHeading/__stories__/LegacyHeading.stories.mdx

```diff
@@ -52,8 +52,8 @@ export const headingTemplate = createComponentTemplate(Heading);
 Heading components are used to title any page sections or sub-sections in top-level page sections.
 
 <Canvas>
-  <Story name="Overview" args={{ value: "Hello world" }}>
-    {headingTemplate.bind({})}
+  <Story name="Overview" args={{}}>
+    <Heading value="Hello world" />
```

Comment:
It means that we can't play with the component in Storybook, isn't it?

Reply:
Yeah, unfortunately :(
But I wasn't able to remove LegacyHeading from the show code other way

Reply:
got ya
actually it might not be that important in a legacy one